lerna.json

```diff
@@ -1,5 +1,7 @@
 {
 	"packages": [ "client", "packages/*", "apps/*" ],
 	"version": "independent",
-	"ignoreChanges": [ "**/*.md", "**/.eslintrc.{js,json,yaml,yml}", "**/package-lock.json" ]
+	"npmClient": "yarn",
+	"useWorkspaces": true,
+	"ignoreChanges": [ "**/*.md", "**/.eslintrc.{js,json,yaml,yml}" ]
```

Comment:
Out of curiosity, why do we remove `package-lock.json` from this list?

Reply:
`package-lock.json` is only relevant when the repo is using `npm`. As we are moving to `yarn`, I deleted that file, so it doesn't make sense to reference it anymore